AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -1652,9 +1652,8 @@ trip101.com##.exit-intent
 sec-ed.co.uk#$#.popupWorks { display: none!important; }
 sec-ed.co.uk#$#.ui-front-modal { display: none!important; }
 sec-ed.co.uk#$#body { overflow: visible!important; }
-aeon.co##div[class^="Banner__BannerWrapper"]
-||psyche.co/_next/static/chunks/SupportBannerHeader.*.js
-||aeon.co/_next/static/chunks/SupportBannerHeader.*.js
+/_next/static/chunks/SupportBannerHeader.*.js$domain=aeon.co|psyche.co
+aeon.co##div[class^="Banner__BannerWrapper-sc-"]
```

Comment:
Already blocked by `aeon.co##div[class^="Banner__BannerWrapper"]` Why replaced?

Reply:
I moved it to colocate all these Aeon/Psyche filters together to make it easier to maintain, but can keep the old one there and remove it from here if you prefer.